tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -112,6 +112,41 @@ class WskBasicUsageTests
         }
     }
 
+    // If client certificate verification is off, should ingore run below tests.
+    if (!WhiskProperties.getProperty("whisk.ssl.client.verification").equals("off")){
+        it should "set valid cert key to get expected success result for client certificate verification" in {
+            val tmpwskprops = File.createTempFile("wskprops", ".tmp")
```

Comment:
* For above, it will send Http request to `wskprops.apihost`.
* For blow set (it should "set multiple property values with single command"), it will send http request to `openwhisk.ng.bluemix.net`, so for here, i think, can move --cert/key, because `openwhisk.ng.bluemix.net` env doesn't support cert/key.
* For below unset ( it should "validate default property values"), just remove --cert/key